### DISCUSSION TITLE
DatePicker Placeholder Text not shown

### DISCUSSION THREAD

**Discussion by MarioXCB** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3330] [Q&A] [answered]
Hello,

i'am using this Toolkit to build an application with many DatePickers. They all have shown the placeholder text in the beginning.
But now the text isn't shown anymore. 
![grafik](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/assets/67013745/664d7200-dacb-4106-943a-83915538354e)

Is there anything I can look for, to fix this error?

**Comment by nicolaihenriksen** [comment]
@MarioXCB Could you elaborate a bit on the issue. When I run the demo app (both latest code and latest release), the hint and helper texts are appearing just fine (see below). Are you possibly using a custom offset?

![image](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/assets/19572699/54f740e7-901a-4ff0-99e7-a806c649f039)

![image](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/assets/19572699/37c813f8-9402-4ac4-9d20-ea628aa11e25)

**Reply by MarioXCB** [reply]
Sure, I can give more information. I testet the DemoApp and it works fine. I think that I've overwritten some property.
These are my custom styles in my Resource.xaml:
```<Style TargetType="{x:Type DatePicker}" BasedOn="{StaticResource MaterialDesignOutlinedDatePicker}">
    <!--<Setter Property="Padding" Value="3" />-->
    <Setter Property="FontFamily" Value="{DynamicResource Roboto}" />
    <Setter Property="FontSize" Value="14" />
</Style>
<Style TargetType="{x:Type DatePickerTextBox}" BasedOn="{StaticResource MaterialDesignDatePickerTextBox}">
    <!--<Setter Property="Padding" Value="0" />-->
    <Setter Property="FontFamily" Value="{DynamicResource Roboto}" />
    <Setter Property="FontSize" Value="14" />
</Style>```

**Reply by MarioXCB** [reply]
And here is an Example of how I'am using the DatePicker component
![grafik](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/assets/67013745/81995bb3-505b-4ba7-b7a3-ddccb590fda2)

**Reply by nicolaihenriksen** [reply, marked as answer]
@MarioXCB Based on what you have provided here, you are not setting the `HintAssist.HelperText` nor the `HintAssist.Hint` attached properties anywhere, which would probably explain why you are not seeing them.

In the image with the XAML content above, you can see how these are set in the XAML for the `DatePicker`.

**Reply by MarioXCB** [reply]
Thank you very much, setting the HintAssist.Hint in my Resource file fixed it
